Lib/test/support/__init__.py

```diff
@@ -532,6 +532,10 @@ def requires_legacy_unicode_capi():
 
 has_fork_support = hasattr(os, "fork") and not is_emscripten and not is_wasi
 
+# From python 3.12.6
```

Comment:
I am sorry to confuse you. Version was required for commit message, but this is also fine too